addons/binding/org.openhab.binding.http/README.md

```diff
@@ -0,0 +1,128 @@
+# HTTP Binding
+
+This binding can be used to make HTTP requests to fetch and update device state.
+
+## Supported Things
+
+This binding supports a single thing type: `http`.
+
+## Discovery
+
+Automatic discovery is not supported.
+
+## Binding Configuration
+
+The binding itself has no configuration.
+
+## Thing Configuration
+
+The Thing itself does not support any configuration.  However, to be useful,
+any number of channels can be added to a created Thing.  Each channel can be
+configured with different URLs to fetch and update state.
```

Comment:
Please remove this section. You can join "Each channel can be configured with different URLs to fetch and update state" with the introduction sentence of the next section, if you want to.